src/service/lootDrop.ts

```diff
@@ -228,12 +230,12 @@ export async function postLootDrop(
 
 type AdjustmentResult = {
     messages: string[];
-    weights: number[];
+    loot: LootTemplate[];
 };
 
-async function getDropWeightAdjustments(
+export async function adjustLootWithBuffsFromUser(
```

Comment:
Der Purpose der Funktion sind generell Adjustments in der Situation des Klickens. Das können Buffs, aber auch Debufffs durch aktuelle Items sein. Später war gedacht, die aktulle Uhrzeit des Klickens oder den schenkenden User ("Schenker X hat Item Y, deshalb ist Warhscheinlichkeit Z höher/geringer") etc.

Reply:
Mit 	"Buffs" meinte ich tatsächlich auch beides. Vielleicht git's da eine bessere Bezeichnung

Reply:
Weight Adjustments